hawk/app/lib/util.rb

```diff
@@ -363,15 +363,43 @@ def acl_version
   end
   module_function :acl_version
 
-  def get_meta_data(agent)
-    res = safe_x("/usr/sbin/crm_resource", "--show-metadata", agent)
+  # Hash#dig from Ruby 2.3
```

Comment:
is there any specific reason why we are monkeypatch a construct from ruby a part of style?

What would be the solution without this?

Reply:
A custom implementation of Hash#dig was added because Hawk's build dependencies described in README.md say that only ruby >= 2.2 is required, but Hash#dig was introduced in version 2.3.